api/src/API/Controllers/MembersController.cs

```diff
@@ -35,34 +37,44 @@ public async Task<IActionResult> GetAllMembers(string orgId)
             return Ok(result);
         }
 
-        [HttpGet("{memberId}")]
-        public async Task<IActionResult> GetOneMember(string orgId, string memberId)
-        {
-            MemberContainerClient container = containerProvider.MemberContainer;
-            Member result = await container.GetOneAsync(memberId, orgId);
-            return Ok(result);
-        }
-
+        [HttpGet("orgAssignedMemberId/{orgAssignedMemberId}")]
```

Comment:
From our conversation earlier today, I think it would be better if we have a query parameter for this operation on the org/id/member endpoint instead of a dedicated route

Reply:
I did that initially, but because the query parameters are optional the route matches GetAllMembers and ASP.NET gets upset about an ambiguous route. I spent hours trying to figure out how to get this working with a param. I don't want to spend more time fighting with ASP.NET.  Feel free to open an issue and investigate later

Reply:
You can merge the two and have the query be an optional parameter. If there's no query, get all. If there is, get the one.

Reply:
No, because GetAll requires authorization whereas this allows anonymous

Reply:
I think I can manually check if the user is authenticated after checking for the query param